mib-generator/cw_mib_generator.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+
+# Copyright (C) Metaswitch Networks 2017
+# If license terms are provided to you in a COPYING file in the root directory
+# of the source code repository by which you are accessing this code, then
+# the license outlined in that COPYING file applies to your use.
+# Otherwise no rights are granted except for those provided to you by
+# Metaswitch Networks in a separate written agreement.
+
+"""Metaswitch Clearwater MIB Generator
+Usage:
+  cw_mib_generator.py [--cwc-mib-dir=DIR]
+
+Options:
+  --cwc-mib-dir=DIR  Directory containing Clearwater Core MIB fragment
+"""
+
+import re
+import sys
+import os.path
+from string import Template
+from docopt import docopt
```

Comment:
I've just tried `autopep8` against the script, and the only thing it diff's is a bit of whitepace in `substitute_dict`. No complaints about the import statements. This seems to go against what you expected?